app/views/layouts/application.html.erb

```diff
@@ -4,6 +4,10 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Timber</title>
+    <link href="https://fonts.googleapis.com/css?family=Raleway|Roboto|Play" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
Do you need this link when we don't make use of icons?

Reply:
Removed unused links.